src/test_driver/nrfconnect/main/runner.cpp

```diff
@@ -29,13 +30,24 @@ using namespace ::chip::DeviceLayer;
 
 LOG_MODULE_REGISTER(runner, CONFIG_MATTER_LOG_LEVEL);
 
-extern "C" int main(void)
+extern "C" {
+void console_init(void) {}
+
+int console_putchar(const char * chr)
+{
+    LOG_RAW(chr);
+    return 0;
+}
+
+int main(void)
 {
     VerifyOrDie(settings_subsys_init() == 0);
 
     LOG_INF("Starting CHIP tests!");
     int status = RunRegisteredUnitTests();
+    status += chip::test::RunAllTests();
```

Comment:
Is the plan to completely replace the old method of registering UTs or we're going to support both in parallel?

Reply:
The plan is to replace the old method completely successively test after test. The old method will be removed at the end of the process.

Reply:
Thanks for the info!